tests/test_convolution.py

```diff
@@ -168,6 +161,39 @@ def _precompute_convolution_tensor_dense(
 
 @parameterized_class(("device"), _devices)
 class TestDiscreteContinuousConvolution(unittest.TestCase):
+    """
+    Test the discrete-continuous convolution module.
+
+    Parameters
+    ----------
```

Comment:
Why do the test classes have docstrings. It is great that the individual members have those but what do the class member docstrings describe? Technically, the constructor for those classes do not get any parameters and those are just to group tests together. That being said, it would be sufficient to have a docstring like

"""

Test the discrete-continuous convolution module.
"""

And then the individual member functions have docstrings describing the individual test.